app/src/api/learningJourneyCourse.js

```diff
@@ -0,0 +1,29 @@
+import axios from "axios";
+import { LEARNING_JOURNEY_COURSE_ENDPOINT } from "./config";
+
+const axiosLJInstance = axios.create({
+    baseURL: LEARNING_JOURNEY_COURSE_ENDPOINT,
+    timeout: 5000,
+    headers: { "X-Custom-Header": "foobar" },
+});
+
+export const getLearningJourneyCoursesById = async (ljId) => {
+    try {
+        const res = await axiosLJInstance.get(`/${ljId}`);
+        if (res) {
+            return extractCourseIds(res.data);
+        }
+        throw new Error("No data returned from backend");
+    } catch (error) {
+        console.log(error);
+        return [];
+    }
+};
+
+function extractCourseIds(ljAndCourseIdsArray) {
```

Comment:
/nit 
1. LJ should be caps. 
2. 2. Can Omit Array from the variable name also i think because if we ever change the schema then it needs to change.  
3. Maybe can name it extractcourseidsfromxxx?

Reply:
done, not sure if the new function name is the most suitable tho